floor/floor/controller/midi/functions.py

```diff
@@ -26,64 +27,94 @@ def add(cls, name, **kwargs):
 
 
 MidiFunctions.add('playlist_next',
+                  callback=lambda controller, _: controller.playlist.next(1),
                   help_text='Advance to the next item in the current playlist.')
 MidiFunctions.add('playlist_previous',
+                  callback=lambda controller, _: controller.playlist.previous(1),
                   help_text='Go to previous item in the current playlist.')
-MidiFunctions.add('playlist_pause',
-                  help_text='Pause the current playlist.')
```

Comment:
I think `pause` was extra.  Don't see a difference between it and `stay`

Reply:
Doh! You're right.